haiku-apps/belife/belife-1.0.0.recipe

```diff
@@ -14,7 +14,9 @@ HOMEPAGE="https://github.com/HaikuArchives/BeLife"
 COPYRIGHT="2006 Studio-33"
 LICENSE="BeLife License"
 REVISION="2"
-SOURCE_URI="$(HOMEPAGE)/archive/96e31e8b4543723c7c8ea4a3d0c57c8cd52f82ca.tar.gz"
+SOURCE_URI="${HOMEPAGE}/archive/96e31e8b4543723c7c8ea4a3d0c57c8cd52f82ca.tar.gz"
+CHECKSUM_SHA256="2566dc527e336277173179c7cce0771579257e19bbcd853377cd6112547a08d1"
+SOURCE_DIR="BeLife-96e31e8b4543723c7c8ea4a3d0c57c8cd52f82ca"
 
 ARCHITECTURES="x86 x86_gcc2"
```

Comment:
If it's a new recipe then I put x86_gcc2 first. Otherwise I leave it the way it is.
In other words, there is no need to change this line if the only change is swapping the order the archictectures appear.